src/reactions/github/issue-assigned.ts

```diff
@@ -0,0 +1,33 @@
+import {
+	Reaction,
+	ReactionHandleOptions,
+	ReactionCanHandleOptions,
+} from './reaction';
+import { IssuePayload } from '../../schemas/github/issue-payload';
+
+export class IssueAssigned extends Reaction {
+	getStreamLabsMessage({
+		payload,
+	}: ReactionHandleOptions<IssuePayload>): string {
+		return `*${payload.sender.login}* has a new assigned issue in *${payload.repository.full_name}*`;
+	}
+	getTwitchChatMessage({
+		payload,
+	}: ReactionHandleOptions<IssuePayload>): string {
+		return `*${payload.sender.login}*  has a new assigned issue in ${payload.repository.html_url}`;
```

Comment:
Shouldn't this be payload.assignee.login ?
Otherwise it would report the login for the person who assigned the issue to the assignee.

Reply:
Good catch!